src/rgw/rgw_rados.cc

```diff
@@ -10779,6 +10781,187 @@ int get_obj_data::get_complete_ios(off_t ofs, list<bufferlist>& bl_list) {
   return 0;
 }
 
+static void _cache_aio_completion_cb(sigval_t sigval);
+
+vector<string> split(const string &s, const char * delim) {
```

Comment:
I think Ceph has a class almost exactly like this

Reply:
Do you know which file I can look in?